### PR TITLE
[CORL-2742] Make dashboard staff counts include admins, mods, site mods, and staff

### PR DESCRIPTION
**Pull request by nick-funk**
## What does this PR do?

Updates dashboard count metrics to include admins, mods, site mods, and staff comment counts in its calculations.

## These changes will impact:

- [ ] commenters
- [X] moderators
- [X] admins
- [ ] developers

## What changes to the GraphQL/Database Schema does this PR introduce?

No changes.

## Does this PR introduce any new environment variables or feature flags? 

No new feature flags.

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- Go to `Admin > Dashboard`
    * Check the amount of `Staff + comments` you have
- Make some staff/admin/mod comments using various roles on a stream
- Go to `Admin > Dashboard`
- Check that comment counts for `Staff + comments` matches the counts you created
 
## How do we deploy this PR?

No special considerations.

### CODE REVIEW

File: src/locales/en-US/admin.ftl [hunk 1]
```diff
@@ -1698,7 +1698,7 @@ dashboard-alltime-rejections-archiveEnabled = { $value } { $unit ->
     }
     *[other] unknown unit
   } average
-dashboard-today-staff-comments = Staff comments
+dashboard-today-staff-plus-comments = Staff + comments
```
Comment (kabeaty) [inline src/locales/en-US/admin.ftl]: Should this be `Staff+ comments` without a space instead of `Staff + comments`?

**Review by kabeaty — CHANGES_REQUESTED** [review]
Looking good overall! I left one comment confirming spacing for the updated `Staff+` copy. 

Noting here for reference as well, but as mentioned, I also noticed while testing that when I leave a comment as an admin and refresh the dashboard as another user, the admin comment doesn't show up in the dashboard comment count until I leave a comment as another staff role (like moderator), and then all comments show correctly and it's incremented for admin as well. When I leave a comment as a moderator and refresh the dashboard as another user, the count increments as expected right away. Can you look into this?

File: src/locales/en-US/admin.ftl (hunk 1 above)
Comment (nick-funk) [reply src/locales/en-US/admin.ftl]: Oh my... I am not going to lie, my brain 100% thought `Staff+` was a typo in the requirements doc and did not think on it one second further. `Staff+` makes so much more sense when you point it out, I will fix this!

Comment (nick-funk) [reply src/locales/en-US/admin.ftl]: Fixed!

**Comment by nick-funk** [comment]
> Looking good overall! I left one comment confirming spacing for the updated `Staff+` copy.
> 
> Noting here for reference as well, but as mentioned, I also noticed while testing that when I leave a comment as an admin and refresh the dashboard as another user, the admin comment doesn't show up in the dashboard comment count until I leave a comment as another staff role (like moderator), and then all comments show correctly and it's incremented for admin as well. When I leave a comment as a moderator and refresh the dashboard as another user, the count increments as expected right away. Can you look into this?

Thank you for testing! I will definitely look into this. Hopefully, it's as simple as a tag not being set or broadcasted properly.

**Comment by nick-funk** [comment]
> Looking good overall! I left one comment confirming spacing for the updated `Staff+` copy.
> 
> Noting here for reference as well, but as mentioned, I also noticed while testing that when I leave a comment as an admin and refresh the dashboard as another user, the admin comment doesn't show up in the dashboard comment count until I leave a comment as another staff role (like moderator), and then all comments show correctly and it's incremented for admin as well. When I leave a comment as a moderator and refresh the dashboard as another user, the count increments as expected right away. Can you look into this?

The admin comment count issue you described is now fixed, I needed to switch to the older cursor iteration format (different from the current mongo docs) as it was only sometimes iterating the cursor correctly.

It should now show correct counts!

**Review by kabeaty — APPROVED** [review]

**Pull request merged**